ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/FilterNodeMapImpl.java

```diff
@@ -67,12 +67,9 @@ public String getOperator() {
 
   @Override
   public List<FilterNode> getChildren() {
-    return Stream.of(json.get(CHILDREN))
-        .map(List.class::cast)
-        .flatMap(List::stream)
-        .map(Map.class::cast)
-        .map(FilterNodeMapImpl::new)
-        .collect(Collectors.toList());
+    List<Object> filters = (List<Object>) json.get(CHILDREN);
```

Comment:
java 11 doesn't like the type errasure that the previous version had. Made it more explicit.